cloud.foundry.cli/src/test/java/cloud/foundry/cli/system/GetAllCommandTest.java

```diff
@@ -0,0 +1,145 @@
+package cloud.foundry.cli.system;
+
+import cloud.foundry.cli.crosscutting.mapping.YamlMapper;
+import cloud.foundry.cli.crosscutting.mapping.beans.*;
+import cloud.foundry.cli.system.util.ArgumentsBuilder;
+import cloud.foundry.cli.system.util.RunResult;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test for {@link cloud.foundry.cli.services.GetController}, specifically the default get command which fetches all
+ * the information and outputs them in the same format we expect for the input.
+ */
+public class GetAllCommandTest extends SystemTestBase {
+
+    @BeforeEach
+    private void cleanUpSpaceBefore() {
+        spaceConfigurator.clear();
+    }
+
+    @AfterEach
+    private void cleanUpSpaceAfterwards() {
+        spaceConfigurator.clear();
+    }
```

Comment:
Doing a cleanup both before and after each test case increases the runtime of the tests quite a bit. We might want to reduce the number of performed cleanups to an amount as small as possible

Reply:
We had it as an `@AfterAll` last evening, but then suspected it to be the reason for the broken log capturing. Turned out it wasn't, so I'll quickly restore that.